tesla_api/energy.py

```diff
@@ -50,10 +56,33 @@ async def get_battery_count(self):
         info = await self.get_energy_site_info()
         return int(info["battery_count"])
 
-
     async def get_energy_site_live_status(self):
         return await self._api_client.get('energy_sites/{}/live_status'.format(self._energy_site_id))
 
+    async def get_energy_site_calendar_history_data(self, kind: str = 'energy', period: str = 'day',
+                                                    end_date: [str, datetime] = None) -> Optional[dict]:
```

Comment:
Is there a case where this is going to return `None`? That's what the `Optional` suggests..